src/components/clusters/ClusterDetailsMultiRegion/components/Overview/SubscriptionSettings/SubscriptionSettings.jsx

```diff
@@ -169,7 +169,7 @@ function SubscriptionSettings({
                     {serviceLevelStr}
                   </DescriptionListDescription>
                   {isEditViewable && (
-                    <DescriptionListDescription class="pf-v6-u-mt-lg">
+                    <DescriptionListDescription className="pf-v6-u-mt-lg">
```

Comment:
This error was causing an error to be displayed in the console when tests were run.